script/firebaseScriptLogWork.js

```diff
@@ -69,7 +69,7 @@ firebase.auth.EmailAuthProvider.PROVIDER_ID
 // privacyPolicyUrl: '<your-privacy-policy-url>'
 };  
 
-function createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category) {
+function createNewTask(title,des,priority,difficulty,creator,assignee,estimatedTime,status,category,loggingDate) {
```

Comment:
It would be better to name it creationDate rather than loggingDate

Reply:
yeah, i will make changes.